aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/question_list.html

```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block faq_footer %}
-    {% url 'aldryn_faq:faq-category-list' as category_url %}
-    {% include "aldryn_faq/includes/pager.html" with title=_("Back to Overview") slug=category_url %}
+    {% url 'aldryn_faq:faq-category-list' as category_list_url %}
+    {% include "aldryn_faq/includes/pager.html" with title=_("Back to Overview") slug=category_list_url %}
```

Comment:
I think we should leave it as slug but as you say add some comments as to what is this expected to do (passing slug)